packages/requisitions/src/SupplierRequisition/ListView/ListView.tsx

```diff
@@ -48,7 +48,7 @@ export const SupplierRequisitionListView: FC = () => {
 
   const columns = useColumns<RequisitionRow>(
     [
-      { ...getNameAndColorColumn(), setter: onUpdate },
+      [getNameAndColorColumn(), { setter: onUpdate }],
```

Comment:
just an observation.. it is possible to update the colour of finalised invoices from the ListView - though you cannot make changes in the DetailView. I think for colour it's probably fine to update finalised invoices, but then the DetailView functionality is inconsistent

Reply:
Yup - similar on the detail views with being able to select rows when finalised..

I'd like your opinion before doing anything about it. 

- If we just disable the row, what do we do with the 'color button' - hard to make it look disabled?
- Make the 'enabled' button look different so that it's easier to make the 'disabled' button look disabled?
- Snack/toast saying 'can't change colour of this row'?

🤔

Reply:
I wasn't sure how to make the colour button look disabled, or how to make it look more enabled! at least, not without making it look clunky. Agree with the underlying concern though, that we need to make the distinction obvious somehow. it would be annoying to have the colour button look the same, but work sometimes and not work for other rows.

Slightly dodgy, but It would be easier to enable the colour change on the detail view. It's worth asking if we should just enable colour change in both. there might be a reason you want to mark specific invoices.. even finalised ones? maybe..?

If that's not ok, I like the toast option - allow people to click the colour button and show a message. I'm usually against that approach; I think if the system knows that I can't perform an action, why does it allow me to do it, and then pop up a dialog telling me? Just disable the action 🤷  The case for using that approach is to give some information, saying why I shouldn't have clicked that button. for the colour button.. the case for doing it, is that it's hard to style the colour button 😁  and maybe also informing the user